andela/activo-api
# #159533673 Refactor Load Method To Allow Partial Loading

#### What does this PR do?
>- Refactors the load method in base schema to allow partial loading
>- Refactors patch endpoints to make use of the functionality
#### Description of Task to be completed?
>- Load method in base schema should allow partial loading, and all update schema classes should be removed and update implementations should be refactored using the partial loading
#### How should this be manually tested?
>- In your terminal, run `git fetch origin ch-refactor-updates-159533673:ch-refactor-updates-159533673`, then run `git checkout ch-refactor-updates-159533673`
>- Run `pytest` to verify that the refactoring did not break the application
#### Any background context you want to provide?
>- Updates were being done by creating a separate schema for the purpose of updating. This PR refactors that implementation.
#### What are the relevant pivotal tracker stories?
>- [#159533673](https://www.pivotaltracker.com/story/show/159533673)
#### Screenshots (if appropriate)
>- N/A
#### Questions:
>- N/A
